jobs/instances-snapshot-cleaner/Dockerfile

```
@@ -0,0 +1,16 @@
+# Using apline/golang image
+FROM golang:1.23-alpine
+
+# Set destination for COPY
+WORKDIR /app
+
+# Copy required files
+COPY go.mod ./
+COPY go.sum ./
+COPY *.go ./
+
+# Build the executable
+RUN go build -o /jobs-snapshot-cleaner
+
+# Run the executable
+CMD [ "/jobs-snapshot-cleaner" ]
```

```suggestion
ENTRYPOINT [ "/jobs-snapshot-cleaner" ]
```
here it won't change anything to the behavior but it just respects more the docker norms, use ENTRYPOINT when we want the container to behave like an executable.